ReactWindows/ReactNative/Bridge/CatalystInstance.cs

```diff
@@ -61,6 +68,22 @@ public ICatalystQueueConfiguration QueueConfiguration
             get;
         } 
 
+        public JavaScriptModulesConfig JSModule
```

Comment:
Delete this property, it is passed into the constructor, it should not be needed on the public surface.